### PR TITLE
Started work on adding MathPower command, see #367

### PR DESCRIPTION
**Pull request by Oire**
@JoshuaKGoldberg, I started to work on #367. It seems, I've done everything I found to add a new command. However, tests error at me and I don't know why. Your help is deeply appreciated. Thanks!

### CODE REVIEW

File: src/Languages/Ruby.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (JoshuaKGoldberg) [inline src/Languages/Ruby.ts]: Sylistic nitpick: unnecessary blank line here (should be removed)

**Comment by Oire** [comment]
Thanks! Definitely missed it, sorry.  
However, it still throws the following (though I didn't change the addition operator at all):

```shell
D:/Repos/Git/gls/test/ComparisonTestsRunner.ts
ERROR: 84:68  restrict-plus-operands  Operands of '+' operation must either be both strings or both numbers
```

**Comment by JoshuaKGoldberg** [comment]
The error in `ComparisonTestsRunner.ts` looks like a bug in TSLint. Very odd... You can ignore it, and run `gulp src:tsc && gulp test` to build+test instead.

Now that you've imported the file in `CommandsBagFactory.ts` you'll also need to add it to the commands being created in the `forContext` method. They're in alphabetical order :)

**Comment by Oire** [comment]
@JoshuaKGoldberg, do I have to change anything else here? Thanks!

**Comment by JoshuaKGoldberg** [comment]
@Oire yes - right now the Travis CI build is failing because of an error in the Python test files. You can run `gulp` locally to verify whether everything's working (see the newly rewritten [docs/development.md](https://github.com/general-language-syntax/GLS/blob/master/docs/development.md)).

Looking through the raw log from https://travis-ci.org/general-language-syntax/GLS/jobs/286767285, it's failing on `test/integration MathPower math power Python` because the test case is missing `from math import pow`.

**Comment by JoshuaKGoldberg** [comment]
LGTM; thanks for the contribution!

**Comment by Oire** [comment]
Thanks a lot! :)

**Pull request merged**